test/runtests.jl

```diff
@@ -1314,3 +1314,85 @@ end
         @test predict(mdl1) ≈ predict(mdl2)
     end
 end
+
+@testset "dropcollinearity in GLM" begin
+    num_rows = 100_000
+    dfrm = DataFrame()
+    dfrm[!, :x1] = randn(StableRNG(123), num_rows)
+    dfrm[!, :x2] = randn(StableRNG(456), num_rows)
+    dfrm[!, :x3]= 2*dfrm[!, :x1] + 3*dfrm[!, :x2]
+    dfrm[!, :y] = Int.(randn(StableRNG(9999999), num_rows) .> 0)
+    @testset "General test" begin
```

Comment:
This testset isn't very useful as it doesn't check many quantities (coefficients, etc.) like the ones below. Better drop the first model completely as it's tested below. The second one should either be dropped or tested against R (and put below).

Also, can you add line breaks before `@testset`?

Reply:
Okay - we will change and update here

Reply:
I have updated the test cases.